### PR TITLE
Makes  all of the genemodded human races actual genemodders now and not custom species.

### PR DESCRIPTION
**Pull request by CrackParrot11**
Makes all of the furry and genemodded races into the Genemodder species option. now you cannot be a sergal and a genemodder that looks like a sergal anymore. You must now pick the Genemodder option to look like a Sergal and the other various human genemodder species as well.

### CODE REVIEW

File: code/modules/species/station/human_subspecies.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/datum/species/human/gravworlder
+//datum/species/human/gravworlder
```
Comment (chaoko99) [inline code/modules/species/station/human_subspecies.dm:1]: You need to use open and closing comments. This isn't how you comment entire types out.

File: code/modules/species/station/human_subspecies.dm [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 	appearance_flags = HAS_HAIR_COLOR | HAS_SKIN_TONE_GRAV | HAS_LIPS | HAS_UNDERWEAR | HAS_EYE_COLOR
 
-/datum/species/human/spacer
+//datum/species/human/spacer
```
Comment (chaoko99) [inline code/modules/species/station/human_subspecies.dm:28]: Same here.

File: maps/torch/torch_ranks.dm [hunk 3]
```diff
@@ -726,8 +726,8 @@
 	pow_cat = 5
 
 /datum/mil_rank/civ/foundationadvisor
-	name = "Psionic Advisor"
-	name_short = "PADV"
+	name = "Foundation Advisor"
```
Comment (chaoko99) [inline maps/torch/torch_ranks.dm:729]: Undocumented.

File: code/modules/species/station/human_subspecies.dm (hunk 2 above)
Comment (chaoko99) [reply code/modules/species/station/human_subspecies.dm:28]: Furthermore, we have version control. Just delete the entire block, honestly.

File: maps/torch/torch_ranks.dm (hunk 3 above)
Comment (chaoko99) [reply maps/torch/torch_ranks.dm:729]: Also I believe in conflict with the existing ruling on the matter.

Comment (CrackParrot11) [reply maps/torch/torch_ranks.dm]: Fixed

File: baystation12.dme [hunk 4]
```diff
@@ -655,7 +655,6 @@
 #include "code\game\jobs\job\science.dm"
 #include "code\game\jobs\job\security.dm"
 #include "code\game\jobs\job\silicon.dm"
-#include "code\game\jobs\job\symbiote.dm"
```
Comment (ProbablyCarl) [inline baystation12.dme:658]: Leave this loaded.

Comment (CrackParrot11) [reply baystation12.dme]: done

**Comment by BurgerLUA** [comment]
Travis failing.

**Comment by Tename** [comment]
Why, this is just going to fuck up peoples existing characters, those species exist because they have developed lore that is supposed to be followed, and there is no reason for this.

**Comment by chaoko99** [comment]
On a server with many, many users. One person playing sergal, for example (Who I believe quit recently) means we have a justification to remove what is honestly a meme race. It's bloat. We don't need bloat.

**Comment by Tename** [comment]
What are you talking about plenty of people play tarajan, vulp, and sergal. All 3 have set lore too which needs a species to properly enforce the lore. Sergal is a bit of a meme and I believe it should have a name change and lore change, but remove several other species with no issues is just a dick thing to do.

**Comment by ProbablyCarl** [comment]
We won't be merging this in the foreseeable future.
Not only will it impact existing characters adversely, but we're seeing an increase of playing the 'removed' species.

**Pull request closed without merging**